website/content/docs/oss/installing/run-and-login.mdx

```diff
@@ -17,16 +17,17 @@ $ boundary dev
 
 Boundary uses a predictable login name (`admin`) and password (`password`) in
 dev mode. These can be overridden, or randomly generated, with flags to
-`boundary dev`.
+`boundary dev`. The auth method resource resides in the global scope and is set
+as the primary for that scope, which means you don't need to specify the
+auth method ID as a flag when using the authenticate command, all you need
+is the short form `boundary authenticate`.
```

Comment:
this could be written a little more clearly:

```suggestion
`boundary dev`. The auth method resource in the global scope is set as the primary auth method for that scope. This allows you to omit the auth method ID when authenticating, enabling the short-form `boundary authenticate`:
```